### PR TITLE
feat: use latest carbon token linter

### PR DESCRIPTION
**Pull request by lee-chase**
- Switches stylelint to use the latest stylelint-plugin-carbon-tokens
- Fixes stylelint script which was not working locally or on circle-ci
- Disables two linter warnings 
- Fixes linter error in _PageHeader.scss
- Ups the local severity as we have no errors
- Includes a looser .stylelint locally for CDAI to allow code port as is.

#### Changelog

Use the following command to list changes or manually list out new, changed and
removed files.

M       .stylelintrc.js
A       .yarn/offline-mirror/stylelint-plugin-carbon-tokens-0.1.1.tgz
M       package.json
A       packages/cdai/.stylelintrc.js
M       packages/cdai/src/components/IdeButton/_IdeButton.scss
M       packages/experimental-package/src/components/PageHeader/_page-header.scss
M       yarn.lock

#### Testing / Reviewing

Ran linters and reviewed output

### CODE REVIEW

File: packages/cdai/.stylelintrc.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+module.exports = {
```
Comment (SimonFinney) [inline packages/cdai/.stylelintrc.js:1]: Is there a script in `cdai/package.json` to run this or will it be covered by `lint:style` in the project's root?

**Review by SimonFinney — APPROVED** [review]
LGTM! I had one thought on the CD&AI stylelint configuration

File: packages/cdai/.stylelintrc.js (hunk 1 above)
Comment (lee-chase) [reply packages/cdai/.stylelintrc.js:1]: It is covered by the root. If you look at the ci-check it shows what would be errors outside of CDAI as warnings

image

**Review by asfordmatt — APPROVED** [review]
LGTM

**Pull request merged**